src/main/java/com/tabnine/TabNineCompletionContributor.java

```diff
@@ -97,8 +98,15 @@ private LookupElementBuilder createCompletion(CompletionParameters parameters, C
                 .withInsertHandler((context, item) -> {
                     int end = context.getTailOffset();
                     TabNineCompletion lookupElement = (TabNineCompletion) item.getObject();
-                    context.getDocument().insertString(end + lookupElement.oldSuffix.length(), lookupElement.newSuffix);
-                    context.getDocument().deleteString(end, end + lookupElement.oldSuffix.length());
+                    try {
+                        if (true) throw new IndexOutOfBoundsException("shai: some error");
```

Comment:
??

Reply:
Also, please provide better message to the exception

Reply:
Pushed this line by mistake.